packages/split-button/src/SplitButton.ts

```diff
@@ -158,47 +152,33 @@ export class SplitButton extends SizedMixin(PickerBase) {
             buttons.reverse();
         }
         return html`
-            ${buttons}
-            <sp-popover
-                open
-                id="popover"
-                @click=${this.onClick}
-                @sp-overlay-closed=${this.onOverlayClosed}
-            ></sp-popover>
+            ${buttons} ${this.renderPopover}
         `;
     }
 
-    protected updated(changedProperties: PropertyValues): void {
-        super.updated(changedProperties);
-        if (changedProperties.has('value')) {
-            this.manageSplitButtonItems();
-        }
+    protected async manageSelection(): Promise<void> {
+        super.manageSelection();
+        this.manageSplitButtonItems();
     }
 
     private async manageSplitButtonItems(): Promise<void> {
-        /* c8 ignore next 3 */
-        if (!this.optionsMenu) {
-            return;
-        }
-        if (this.optionsMenu.menuItems.length) {
+        if (this.menuItems.length) {
             if (this.type === 'more') {
-                this.optionsMenu.menuItems[0].hidden = true;
-                this.optionsMenu.menuItems.forEach(
-                    (el) => (el.selected = false)
-                );
-                this.optionsMenu.menuItems[0].selected = true;
-                this.selectedItemText = this.optionsMenu.menuItems[0].itemText;
+                this.menuItems[0].hidden = true;
+                this.menuItems.forEach((el) => (el.selected = false));
+                this.menuItems[0].selected = true;
+                this.selectedItemText = this.menuItems[0].itemText;
             } else {
                 const selected =
-                    this.optionsMenu.menuItems.find((el) => el.selected) ||
-                    this.optionsMenu.menuItems[0];
+                    this.menuItems.find((el) => el.selected) ||
+                    this.menuItems[0];
                 selected.selected = true;
                 this.selectedItemText = selected.itemText;
             }
             return;
         }
-        await this.optionsMenu.updateComplete;
-        if (this.optionsMenu.menuItems.length) {
+        await this.updateComplete;
```

Comment:
If we can actually move away from the use of `this.optionsMenu.menuItems` (see https://github.com/adobe/spectrum-web-components/pull/1237/files#r586106780 for the logic on why we might not be able to) then we shouldn't get anything useful from the `await` here and can remove.